File: cn/java-spring-demo/src/main/java/com/itangcent/springboot/demo/common/model/TestLogBean.java
package com.itangcent.springboot.demo.common.model;

/**
 * 测试日志Bean类
 * 用于演示transient关键字的使用
 * 包含敏感信息处理示例
 */
public class TestLogBean {
    /**
     * 用户名
     * 正常序列化和持久化
     */
    private String userName;

    /**
     * 密码
     * 使用transient修饰，不参与序列化
     * 用于保护敏感信息
     */
    private transient String password;

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
} 

File: cn/kotlin-spring-demo/src/main/kotlin/com/itangcent/common/model/TestLogBean.kt
package com.itangcent.common.model

/**
 * 测试日志Bean类
 * 用于演示@Transient注解的使用
 * 包含敏感信息处理示例
 */
data class TestLogBean(
    /**
     * 用户名
     * 正常序列化和持久化
     */
    var userName: String? = null,

    /**
     * 密码
     * 使用@Transient标记，不包含在序列化中
     * 用于保护敏感信息
     */
    @Transient
    var password: String? = null
) 